fiftyone/utils/labels.py

```
@@ -262,18 +285,33 @@ def import_segmentations(
+                if label.mask_path is not None:
+                    del_path = label.mask_path if delete_images else None
+                    label.import_mask(update=update)
                     if del_path:
                         etau.delete_file(del_path)
-            else:
+            elif isinstance(label, fol.Detection):
                 if label.mask_path is not None:
                     del_path = label.mask_path if delete_images else None
                     label.import_mask(update=update)
                     if del_path:
                         etau.delete_file(del_path)
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)